### ISSUE TITLE
Update app configuration to use new app context

### ISSUE THREAD

**Issue by bferguso** [bcgov/BCHeritage#908]
This is for file-based configuration. Database configuration updates are handled in a separate issue.

#910 must be completed prior to this

**Comment by bferguso** [comment]
This is currently already done in dev. Will also change deployment pattern to centralize configuration and secrets using .env pattern.

**Comment by bferguso** [comment]
This is complete and now in PROD
